doc/manual/src/glossary.md

```diff
@@ -86,6 +86,13 @@
     the same output. This cannot be guaranteed in general (e.g., a
     builder can rely on external inputs such as the network or the
     system time) but the Nix model assumes it.
+    
+  - Nix database{#gloss-nix-database}\
+    An SQlite database to track references between store objects.
```

Comment:
```suggestion
    An SQlite database to track references between store objects in a [local store][store].
```

This is an implementation detail of the local store and that should at least be made clear.